addons/mail/static/src/messaging/component/thread_typing_icon/thread_typing_icon.js

```diff
@@ -0,0 +1,32 @@
+odoo.define('mail.messaging.component.ThreadTypingIcon', function (require) {
+'use strict';
+
+const { Component } = owl;
+
+class ThreadTypingIcon extends Component {}
+
+Object.assign(ThreadTypingIcon, {
+    defaultProps: {
+        animation: 'none',
+        size: 'small',
+    },
+    props: {
+        animation: {
+            type: String,
+            validate: prop => ['bounce', 'none', 'pulse'].includes(prop),
+        },
+        size: {
+            type: String,
+            validate: prop => ['small', 'medium'].includes(prop),
+        },
+        title: {
+            type: String,
+            optional: true,
```

Comment:
Seems like hiding a bug and title should always be set, but fine by me to keep it like this for now.

Reply:
Not bug hiding with this, this props allow setting a title attribute on DOM root node of this component or not, at the appreciation of developer.

Reply:
ok